.github/workflows/ruby.yml

```diff
@@ -36,6 +36,11 @@ jobs:
             bootstrap_version: ~> 4.0
             additional_engine_cart_rails_options: -a propshaft
             additional_name: '/ Propshaft'
+          - rails_version: '7.0.8'
```

Comment:
Is 7.1.3 too high?

Reply:
Yes https://github.com/projectblacklight/spotlight/blob/main/blacklight-spotlight.gemspec#L51